gradle/test.gradle

```diff
@@ -34,7 +34,7 @@ dependencies {
     integrationTestImplementation group: 'org.slf4j', name: 'slf4j-log4j12', version: slf4jVersion
 
     // For Topic name validation based on Kafka classes
-    testImplementation (group: 'org.apache.kafka', name: 'kafka_2.11', version: kafkaVersion) {
+    testCompile (group: 'org.apache.kafka', name: 'kafka_2.11', version: kafkaVersion) {
```

Comment:
Why not test implementation?

Reply:
Otherwise, the integrationTest fails. With a class def not found error for `SecurityProtocol`.